src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -271,6 +279,23 @@ export default class FilterOptionsComponent extends Component {
     if (this.config.storeOnChange) {
       this.apply(this.config.isDynamic);
     }
+
+    if (!this.config.isDynamic) {
+      this.core.globalStorage.on('update', this.name, (data) => {
+        let previousOptions;
+        try {
+          previousOptions = JSON.parse(data);
```

Comment:
If `data` is passed via an `update` listener, wouldn't it be the currently selected options?

Reply:
Yeyeye, I used `previousOptions` as that was how it was described in the function `setSelectedOptions` (and made sense for my case of back nav). But! I think that could be more generalized as `optionsOverrides`. Made that change, considering updates would not /only/ be from a back nav (although I just did a check and am noticing that the only way this specific filter option would be updated currently is through a back nav)

Reply:
Updated the README! Lmk what you think